seqr/urls.py

```diff
@@ -269,6 +271,7 @@
     'matchmaker/v1/match': external_api.mme_match_proxy,
     'matchmaker/v1/metrics': external_api.mme_metrics_proxy,
 
+    'anvil_workspace/(?P<workspace_name>[^/]+)': anvil_workspace_page,
```

Comment:
Can I add the endpoint to the `urlpatterns` like the `login` and `logout` endpoints? Users have to provide both `name` and `namespace` for the workspace. We have to options to do that 1) use the Django URL parameter like `^anvil_workspace/(?P<namespace>[^/]+)/(?P<name>[^/]+)`; 2) use queries in the URL such as `anvil_workspace/?namespace=user_provided&name=user_provided`. Which one do you prefer?

Reply:
> Can I add the endpoint to the `urlpatterns` like the `login` and `logout` endpoints? 

Yes, although it would be more straightforward to add it to the initial `urlpatterns` list like the `status_view`

> Users have to provide both `name` and `namespace` for the workspace. We have to options to do that 1) use the Django URL parameter like `^anvil_workspace/(?P<namespace>[^/]+)/(?P<name>[^/]+)`; 2) use queries in the URL such as `anvil_workspace/?namespace=user_provided&name=user_provided`. Which one do you prefer?

I prefer option 1, as I think it is more explicit and requires less validation (no need to check that all the required query params are there). You should also then make sure the `/create_project_from_workspace` page also has the same url pattern with both the namespace and name in the url